docs/roadmap.md

```diff
@@ -1,9 +1,9 @@
 # Roadmap
 
-This is the official roadmap for the Open Service Broker for Azure (OSBA) project. OSBA is currently available as an alpha release and this roadmap represents our plan to deliver (OSBA) and the services it provides in a stable form.
+This is the official roadmap for the Open Service Broker for Azure (OSBA) project. OSBA is currently available as an stable release and this roadmap represents our plan to continue to evolve (OSBA) and the services it provides.
 
-OSBA releases follow [semantic versioning](https://semver.org/) strictly, and the current stability is pre-`v1.0.0`, which means
-that backward incompatible changes are possible.
+OSBA releases follow [semantic versioning](https://semver.org/) strictly, and the current stability is `v1.0.1`, which means
+that we will not make backward incompatible changes within a 1.x.x release.
```

Comment:
"current stability" reads oddly.

Can we change this whole sentence?

I propose:

> OSBA releases follow [semantic versioning](https://semver.org/) strictly, which means we will not implement backward incompatible changes in a minor or patch release.